Ext4Pkg/Ext4Dxe/File.c

```diff
@@ -340,8 +338,9 @@ Ext4ReadSymlink (
   }
 
   *Symlink = Symlink16Tmp;
-
-  return EFI_SUCCESS;
+  
+  FreePool (SymlinkTmp);

```

Comment:
Seriously, you're micro-optimizing firmware filesystem code?

Reply:
I follow suggestions